spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/statsd/StatsdProperties.java

```diff
@@ -59,7 +59,8 @@
 
 	/**
 	 * How often gauges will be polled. When a gauge is polled, its value is recalculated
-	 * and if the value has changed, it is sent to the StatsD server.
+	 * and if the value has changed (or {@link #publishUnchangedMeters} is true),
```

Comment:
You can't use javadoc tags in here. This is used as the description in IDEs and other metadata clients.